packages/blockchain-link/src/workers/utils.ts

```diff
@@ -20,3 +22,25 @@ export const prioritizeEndpoints = (urls: string[]) =>
         })
         .sort(([, a], [, b]) => b - a)
         .map(([url]) => url);
+
+export const getSolanaStakingAccounts = async (descriptor: string, isTestnet: boolean) => {
+    const networkConfig = {
+        devnet: {
+            network: SolConstants.Network.Devnet,
+            url: 'https://solana-dev.trezor.io/',
+        },
+        mainnet: {
+            network: SolConstants.Network.Mainnet,
+            url: 'https://solana1.trezor.io/',
```

Comment:
Would be better to pull these from blockchain-link `config.ts`
We actually changed these recently to sol1.trezor.io to separate the traffic after our optimizations.

Reply:
I think it should always use the backend set in `wallet.blockchain[symbol].url` state

Reply:
I'm taking the first link from the array for now. Not sure how to get the same URL as in `wallet.blockchain[symbol].url`
 https://github.com/trezor/trezor-suite/pull/15409/commits/8143354243fc6c1a337c480b5852f01a90121c61